numbast/README.md

```diff
@@ -2,16 +2,27 @@
 
 ## Overview
 
-Numbast is the auto binding generator for CUDA C++ headers. It consumes parsed headers from `AST_Canopy` and dynamically create new types, data models and lowering for C++ types. A very high level view of Numbast's future role is that it aims to become a C++-Python syntax translator.
+Numbast is the auto binding generator for CUDA C++ headers. It consumes parsed headers from `AST_Canopy` and dynamically create new types, data models and lowering for C++ types. One could see Numbast as a C++-Python syntax translator.
+
+## Supported CUDA C++ declarations
+
+- Concrete Struct / Classes (e.g. `struct Foo`)
+    - Constructors e.g. `Foo()`
+    - Conversion Operators e.g. `operator float()`
+    - Attribute Read Access e.g. `Foo().x`
+- Concrete Functions `e.g. bar() {}`
```

Comment:
We should probably give an example for each item to how they are translated into in Python syntaxes.